From 8589b44c573d4228a9df3ba7bffdfdf584e7b033 Mon Sep 17 00:00:00 2001
From: Peter Korsgaard <peter@korsgaard.com>
Date: Mon, 9 Jan 2017 21:38:29 +0100
Subject: [PATCH] configure.ac: relax linux OS detection

Not all linux hosts match the *-pc-linux-gnu wildcard, causing build
failures for older glibc versions where we need to link with -lrt for
clock_gettime - E.G.:

 - arm-unknown-linux-musleabihf
 - powerpc-unknown-linux-gnuspe
 - bfin-linux-linux-uclibc
 ..

Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 070ab357..235d2717 100644
--- a/configure.ac
+++ b/configure.ac
@@ -186,7 +186,7 @@ esac
 AM_CONDITIONAL(OS_IS_WINDOWS, test "x$os_is_windows" = xyes)
 
 case "$host" in
-	*-pc-linux-gnu)
+	*-linux-*)
 		sys_linux=true
 		AC_DEFINE(FLAC__SYS_LINUX)
 		AH_TEMPLATE(FLAC__SYS_LINUX, [define if building for Linux])
-- 
2.11.0

